src/lang.js

```diff
@@ -27,7 +27,7 @@
     // Default options //
 
     var defaults = {
-        defaultLocale: 'en' /** The default locale if not set. */
+        defaultLocale: 'en', /** The default locale if not set. */
```

Comment:
Good @rmariuzzo, I was testing returning an object with trailing comma in http://jshint.com/ and its pass without not errors, why? that's strange.